source/navitiacommon/navitiacommon/__init__.py

```diff
@@ -26,3 +26,7 @@
 # IRC #navitia on freenode
 # https://groups.google.com/d/forum/navitia
 # www.navitia.io
+
+from navitiacommon import log
+
+log.enable_color(False)
```

Comment:
I don't see why importing `navitiacommon` should have a side effect on the logging.
Define your function then call it in the projects that need it.

Reply:
The idea was to enable this feature for all python projects so, everytime we import "navitiacommon" (I believe it's done by every python project) we set up the log. It only eneable the format, colors have to be enabled if the program needs it.

Reply:
Removed